packages/commonwealth/server/routes/getAddressProfile.ts

```diff
@@ -29,110 +28,46 @@ type GetAddressProfileResp = {
   avatarUrl: string;
 };
 
-const getAddressProfile = async (
+const getAddressProfiles = async (
```

Comment:
What changes were made to this route? Can you document them?

Reply:
changes:
* Change input format
* Consolidate db queries
* Code cleanup

#### Change input format
The input had changed slightly. Instead of passing address parameters like:
address=firstValue&address=secondValue
I had changed the format into:
addresses=firstValue,secondValue

This is because the extra characters reduce the amount of addresses we can query by (because we were hitting the max body size limit).

####  Consolidating db queries
The other change that was made was to consolidate some db queries into using a joined query.

#### Code cleanup
Before we had it so that requests with only one address were handled differently than with multiple addresses. This was unnecessary and cause a lot of code duplication. As a result I had consolidated these two logic branches into one to remove duplicate code.